config-toml.md

```diff
@@ -0,0 +1,40 @@
+# Config.toml breakdown
+
+- [Config.toml breakdown](#configtoml-breakdown)
+  - [Pre-Requisites](#pre-requisites)
+  - [Understanding config.toml](#understanding-configtoml)
+    - [[Core]](#core)
+    - [[P2P]](#p2p)
+      - [Bootstrap](#bootstrap)
+      - [Mutual Peers](#mutual-peers)
+    - [[Services]](#services)
+      - [TrustedHash and TrustedPeer](#trustedhash-and-trustedpeer)
+
+## Pre-Requisites
+Please, make sure that you have installed and initialized celestia node
+
+## Understanding config.toml
+After initialization, for any type of node, you will find a `config.toml` in path(default location)
+- `~/.celestia-bridge/config.toml` for bridge node
+- `~/.celestia-light/config.toml` for light node 
+
+Let's break down some of the most used sections
+### [Core]
+This section is needed for the Celestia Bridge Node. By default, `Remote = false`. Still for devnet, we are going to use the remote core option and this can also be set
+by the command line flag `--core.remote` 
+
+### [P2P]
+#### Bootstrap
+By default, the `Bootstrapper = false` and the `BootstrapPeers` is empty. If you want your node to be a bootstrapper, then activate `Bootstrapper = true`.
```

Comment:
The a newbie it's completely opaque in what circumstances they would want to be a bootstrap peer.